src/views/CVERecord/SearchResults.vue

```diff
@@ -0,0 +1,300 @@
+                            </span>
+                          </button>
+                        </li>
+                        <li v-for="page in cveListSearchStore.pagination.currentPageWindow" :key="page.key">
+                          <button
+                            @click="cveListSearchStore.handlePageChange(page)"
+                            :class="{'is-current': page === cveListSearchStore.pagination.currentPage}"
+                            class="pagination-link button cve-button cve-button-outline" :aria-label="`Go to page ${cveListSearchStore.pagination.currentPage}`">
+                            {{ page }}
+                          </button>
+                        </li>
+                        <li>
+                          <button @click=cveListSearchStore.handleNextWindowChange
+                            :disabled="cveListSearchStore.pagination.endWindow < cveListSearchStore.pagination.totalPages ? false : true"
+                            :aria-disabled="cveListSearchStore.pagination.endWindow < cveListSearchStore.pagination.totalPages ? false : true"
+                            class="pagination-next button cve-button cve-button-outline"
+                          >
+                            <span class="icon">
+                              <p id="nextWindowIcon" class="is-hidden">next page window button</p>
+                              <font-awesome-icon icon="angles-right" aria-labelledby="nextWindowIcon" aria-hidden="false"/>
+                            </span>
+                          </button>
+                        </li>
+                      </ul>
+                    </nav>
+                  </div>
+                  <div v-else-if="cveListSearchStore.totalSearchResultCount === 0 && cveListSearchStore.query.length === 0" class="has-text-centered">
+                    <p>Please provide your search keywords and select the "Search" button.</p>
+                  </div>
+                  <div v-else>
+                    <p class="has-text-weight-bold">No search results found for: <span class="is-italic">{{ cveListSearchStore.query }}</span></p>
+                    <p>Please try your search again using the following
+                      <router-link to="/ResourcesSupport/FAQs#pc_cve_list_basicssearch_cve_list_tips">search tips and ticks</router-link> on the FAQs page.
+                    </p>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </main>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup>
+import { useCveListSearchStore } from '@/stores/cveListSearch';
+import { usePartnerStore } from '@/stores/partners';
+import { watch } from 'vue';
+import { useRouter } from 'vue-router';
+const cveListSearchStore = useCveListSearchStore();
+const router = useRouter();
+const partnerStore = usePartnerStore();
```

Comment:
Since `usePartnerStore()`  is called directly in other parts of the file, we don't need the `partnerStore` const